src/Application/Kernel.php

```diff
@@ -54,30 +53,12 @@ private function initContainer(): Container
             ->addInitializer(new RouteBindingInitializer())
             ->addInitializer(new PDOInitializer());
 
-        return $container;
-    }
-
-    private function initDiscoveryLocations(): void
-    {
-        $this->discoverTempestNamespaces();
-        $this->discoverInstalledPackageLocations();
-    }
-
-    private function initConfig(Container $container): void
-    {
-        // Register AppConfig
-        $container->config($this->appConfig);
-
-        // Scan for package config files
-        foreach ($this->appConfig->discoveryLocations as $package) {
-            $configFiles = glob(path($package->path, 'Config/**.php'));
-
-            foreach ($configFiles as $configFile) {
-                $configFile = require $configFile;
-
-                $container->config($configFile);
-            }
+        /** @var class-string<\Tempest\Bootstraps\Bootstrap> $bootstrap */
+        foreach ($this->bootstraps as $bootstrap) {
```

Comment:
Why is this necessary? The container should resolve those classes and their dependencies when you call `get` above.

Reply:
It was among my comments in the issue. The response pointed out that i wouldn't need to pass `$container` to `boot()`, but didn't speak to removing this loop registering the singletons.

You're right, of course, it works without, so i'll remove this.